Libplanet/Net/Swarm.TrustedStateCompleter.cs

```diff
@@ -59,6 +59,9 @@ public partial class Swarm<T>
 
             IReadOnlyList<BoundPeer> trustedPeers =
                 (await DialToExistingPeers(dialTimeout, cancellationToken))
+                    .Where(pair =>
+                        !(pair.Peer is null || pair.ChainStatus is null) &&
+                        pair.ChainStatus.GenesisHash.Equals(BlockChain.Genesis.Hash))
```

Comment:
I guess it's enough... (or, Can `pari.Peer` be `null`?) 

```suggestion
                    .Where(pair =>
                        BlockChain.Genesis.Hash.Equals(pair.ChainStatus?.GenesisHash))
```

Reply:
Because of [this line](https://github.com/planetarium/libplanet/blob/e61fd8c35fc0b818d7661cb0a7100624bb354cfb/Libplanet/Net/Swarm.cs#L1389) (Swarm.cs L1389), it can be null.

https://github.com/planetarium/libplanet/blob/e61fd8c35fc0b818d7661cb0a7100624bb354cfb/Libplanet/Net/Swarm.cs#L1389

Reply:
Amended in [`6dd6943`](https://github.com/planetarium/libplanet/pull/1004/commits/6dd69434aff0dae74c49149c90e2bcb331c1b488).